src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -2361,6 +2361,11 @@
         <summary>The input type to the pattern-matching operation.</summary>
       </Comments>
     </Property>
+    <Property Name="OutputType" Type="ITypeSymbol">
```

Comment:
I'll be honest: I don't know what this means. What concept is this actually trying to capture?

Reply:
this is spec'ed as the "narrowed type" or "output type" in pattern matching draft specification. it's most significant in `and` patterns where the output type of the LHS is the input type of the RHS. eg `obj is int and > 6`.

Reply:
That definition uses the term I'm confused on, so it doesn't really help :). Internally, we call this converted type. Is the type that the pattern input will be converted to? If so, `ConvertedType` is a significantly better name. The docs should also be more specific.

Reply:
Sorry the name "output type" is not mentioned at all, I must misread something the first time.

Renamed.